src/main/java/gov/usgs/wma/waterdata/collections/CollectionParams.java

```diff
@@ -65,20 +65,20 @@ public Builder builder() {
 		private List<String> counties;
 		private List<String> states;
 		private List<String> hydrologicUnits;
-		private String nationalAquiferCode;
-		private String agencyCode;
+		private List<String> nationalAquiferCodes;
+		private List<String> agencyCodes;
 		private List<String> monitoringLocationType;
-		private String monitoringLocationNumber;
+		private List<String> monitoringLocationNumbers;
 		private boolean isPaging = false;
-	        private String filterOptions = "";
-
-        	public Builder(ConfigurationService configurationService) {
-        		this.configurationService = configurationService;
+		private String filterOptions = "";
+		
```

Comment:
being too picky, extra s space.

Reply:
Sorry ... where?  I can't see it.

Reply:
I think I misread the diff of a tab changing to space. it's good.